charts/px-backup/templates/px-backup/pxcentral-backup.yaml

```diff
@@ -0,0 +1,126 @@
+{{/* Setting defaults if they are omitted. */}}
+{{- $customRegistryEnabled := .Values.customeRegistryEnabled | default false }}
+{{- $pxBackupEnabled := .Values.pxbackup.enabled | default false }}
+{{- if eq $pxBackupEnabled true }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: px-backup-account
+  namespace: {{ .Values.namespace }}
+  labels:
+{{- include "px-backup.labels" . | nindent 4 }}
+---
+kind: ClusterRole
```

Comment:
We only need permission within the namespace we are installing in, so change the ClusterRole/ClusterRoleBinding to Role/RoleBinding

Reply:
Changed to role and rolebinding